crates/core/src/runtime/runtime.rs

```diff
@@ -7,6 +7,14 @@ use crate::model::{Object, ObjectView, Scalar, ScalarCow, Value, ValueCow, Value
 use super::PartialStore;
 use super::Renderable;
 
+/// What mode to use when rendering.
+pub enum RenderingMode {
```

Comment:
I feel like this should have `Copy`, `Clone`, `Debug`, `PartialEq`, `Eq`, and `Default`